### PR TITLE
image hydration

### PR DESCRIPTION
**Pull request by vvetsa**

### CODE REVIEW

**Comment by vvetsa** [comment]
image
image

File: lib/main_home_page.dart [hunk 1]
```diff
@@ -150,9 +177,11 @@ class NewsCardState extends State<NewsCard> {
     //var newsInfoCard = fetchNewsInfoCardFeed("1");
 
     if (data.isNotEmpty) {
+      //print(data.length);
```
Comment (aiswaryasankar) [inline lib/main_home_page.dart:180]: Remove comments plz

File: lib/main_home_page.dart [hunk 2]
```diff
@@ -309,7 +342,14 @@ class NewsCardState extends State<NewsCard> {
                           setUserEngagementButton('1',
                               'b763d796-4cef-11ed-8f64-acde48001122', 'right');
                           setState(() {
+                            setState(() {
```
Comment (aiswaryasankar) [inline lib/main_home_page.dart:345]: double check this?

File: lib/main_home_page.dart [hunk 3]
```diff
@@ -150,9 +177,11 @@ class NewsCardState extends State<NewsCard> {
     //var newsInfoCard = fetchNewsInfoCardFeed("1");
```
Comment (aiswaryasankar) [inline lib/main_home_page.dart:177]: Clean up all commented out code plz

**Review by aiswaryasankar — APPROVED** [review]

**Pull request merged**